src/types/common.ts

```diff
@@ -148,3 +155,12 @@ export interface BatchSuccessResponse<T> {
 
 export type BatchResponse<T> = BatchErrorResponse | BatchSuccessResponse<T>;
 export type GenericFunction = (...args: any) => any
+
+export type MinMaxLatLngBox = {
```

Comment:
This type should be called `Rectangle` or `Box`, even though it might not always have right angles it would still be more explanatory than this.